arrow-core/src/main/kotlin/arrow/core/extensions/const.kt

```diff
@@ -32,19 +31,16 @@ import arrow.typeclasses.TraverseFilter
 import arrow.core.ap as constAp
 import arrow.core.combine as combineAp
 
-@extension
 interface ConstInvariant<A> : Invariant<ConstPartialOf<A>> {
```

Comment:
Do we need to deprecate all these interfaces as well?

Reply:
Yes, we need to deprecate all these as well.
I realised we've not done this for the previously merged data types, so we should add it for them as well. So not a blocker for this PR for me.

Reply:
I'll add these deprecations!! 👍